pkg/kubectl/cmd/patch.go

```diff
@@ -146,7 +155,8 @@ func RunPatch(f cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []strin
 	r := f.NewBuilder().
 		Unstructured().
 		ContinueOnError().
-		NamespaceParam(cmdNamespace).DefaultNamespace().
+		NamespaceParam(cmdNamespace).DefaultNamespace().AllNamespaces(allNamespaces).
+		SelectAllParam(allInNamespace || allNamespaces).
```

Comment:
This means `--all-namespaces` will automatically include all resources, including un-initialized ones. Right?

IMO, should be `SelectAllParam(allInNamespace)` instead.

Reply:
I’ll have to look again but I think the params are broken.